src/settings.rs

```diff
@@ -46,10 +48,8 @@ pub(crate) struct Keyless {
 pub(crate) struct GithubActions {
     /// String pointing to the object (e.g.: `registry.testing.lan/busybox:1.0.0`)
     pub(crate) image: String,
-    /// owner of the repository. E.g: octocat
-    pub(crate) owner: String,
-    /// Optional - Repo of the GH Action workflow that signed the artifact. E.g: example-repo
-    pub(crate) repo: Option<String>,
+    // GitHub Actions information that must be found in the signature
+    pub(crate) github_actions: KeylessGithubActionsInfo,
```

Comment:
This is really a matter of personal taste, what is the advantage of this change? What does having another extra struct give us?

Just to be clear, we can go ahead with this change, I don't mind.

Reply:
I see, I like it. Go ahead with that!